apps/cowswap-frontend/src/modules/swap/services/swapFlow/index.ts

```diff
@@ -84,6 +86,8 @@ export async function swapFlow(
       ? Promise.resolve(null)
       : presignOrderStep(orderId, input.contract))
 
+    cowEventEmitter.emit(CowEvents.ON_POSTED_ORDER, { orderUid: orderId, chainId: input.context.chainId })
```

Comment:
This is at the right location, after we receive the response from the wallet 👍

Reply:
Thanks for checking. I guess when we do the end-to-end test (and we connect the toasts out of the widget), we will make sure all toasts and messages trigger in the right time